docs/docs/guides/exporting-logs.md

```diff
@@ -404,12 +404,112 @@ kurtosis-clusters:
             codec: "json"
 ```
 
+### Configuring Log Filters
+
+Kurtosis allows you to configure filters that get passed through to the logs collector. These filters can be used to modify or filter logs before they are sent to the configured sinks from export logs.
```

Comment:
Maybe worth specifying upfront "through to the Fluentbit logs collector" (since we're already tied to the implementation)